src/providers/config/config.ts

```diff
@@ -222,7 +223,8 @@ export class ConfigProvider {
         buycrypto: true,
         exchangecrypto: true,
         giftcards: true,
-        walletConnect: false
+        walletConnect: false,
```

Comment:
Then we could try to remove old/unused property

Reply:
It's necessary for backward compatibility. It has a scheme (data type) that could fail if it read a different object.